datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py

```diff
@@ -317,9 +322,9 @@ def generate_trap_db(compiled_mibs, compiled_mibs_sources, no_descr):
 
 
 @lru_cache(maxsize=None)
-def get_oid_and_descr(var_name, mib_name, search_locations=None):
+def get_metadata(var_name, mib_name, search_locations=None):
```

Comment:
Maybe this function should return a single object now (like a namedtuple or some custom class)?
Also what do you think of a slightly more explicit name like `get_var_metadata`?